hearbaseline/keras/naive.py

```diff
@@ -102,6 +104,7 @@ def get_timestamp_embeddings(
     Args:
         audio: n_sounds x n_samples of mono audio in the range [-1, 1].
         model: Loaded model.
+        hop_size: Hop size in milliseconds.
```

Comment:
Make a note here that this is NOT a requirement made by the API, but we include here internally to improve the efficiency of the scene embeddings and something we recommend?